src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private Map<String, Integer> storage = new HashMap<>();
+
+    public void addFruit(String fruit, int quantity) {
+        storage.put(fruit, storage.getOrDefault(fruit, 0) + quantity);
+    }
+
+    public void removeFruit(String fruit, int quantity) {
+        if (storage.containsKey(fruit)) {
+            storage.put(fruit, storage.get(fruit) - quantity);
+        }
+    }
+
+    public Map<String, Integer> getStorage() {
+        return new HashMap<>(storage);

```

Comment:
The getStorage method exposes the internal storage structure. Consider returning an unmodifiable view of the map to follow the encapsulation principle.